pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,99 @@
+// ------------------------------
+// edX Pattern Library: Components - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// ----------------------------
+// #SETTINGS
+// ----------------------------
+@mixin alert-base($alert-color) {
```

Comment:
NIT: since this is really just a mixin or an archetype we're creating alert styles from, I think the mixin name can be simplified to just be ``alert()``.